reference/content-types/text_line.rst

```diff
@@ -20,4 +28,7 @@ Example
         <meta>
             <title lang="en">Title</title>
         </meta>
+        <params>
+            <param name="headline" value="true"/>
```

Comment:
tag's seem like something which can be put on different content-types. The headline parameter only has effects on the text line. Its so to say a parameter of the text line. Because in other content types, like the text-editor or the media-selection, parameters were used in the same sense, @chirimoya told me to implement it as a parameter